plugins/TimingHardwareManagerPDII.hpp

```diff
@@ -70,7 +70,7 @@ class TimingHardwareManagerPDII : public TimingHardwareManager
   TimingHardwareManagerPDII& operator=(TimingHardwareManagerPDII&&) =
     delete; ///< TimingHardwareManagerPDII is not move-assignable
 
-  void conf(const nlohmann::json& conf_data) override;
+  void conf(const nlohmann::json&) override;
```

Comment:
If you remove the variable name of the json& argument from the function definition in the .cpp file instead of from the .hpp file you will a) not get warnings about unused variables when you don't access the json and b) get an error if you do still try and access it as above.

Reply:
I've added the variable name back to the `.hpp` file but it needs to stay in the `.cpp` file as the `TimingHardwareManager::conf` needs the json data to run `configure_uhal`... maybe this is something I can change after this first PR goes in?